src/lib/apiTypesV2/Entity.cpp

```diff
@@ -199,10 +199,24 @@ std::string Entity::check(ApiVersion apiVersion, RequestType requestType)
     return std::string(errorMsg);
   }
 
-  if (forbiddenIdChars(apiVersion, id.c_str()))
+  if (isPattern == "")
```

Comment:
Although we have an issue for an overall fix for this (https://github.com/telefonicaid/fiware-orion/issues/2791) let's try to start using `empty()` in new/modified code from now on.